docs/user_manual/examples/boat_animation.py

```diff
@@ -1,5 +1,4 @@
-import numpy as np
-from numpy import pi as π
+from numpy import pi as pi
```

Comment:
I liked my fancy π! But okay, I understand that it may be causing issues in some setup.
I guess `as pi` is not necessary if we don't rename it.

Reply:
> I guess `as pi` is not necessary if we don't rename it.

Ehm, yes, of course... I replaced all `π` with `pi`